### PR TITLE
French translations and support

### PR DESCRIPTION
**Pull request by ClementMR**
- [x] This PR has been tested locally

Add french translations

### CODE REVIEW

**Comment by LoneWolfHT** [comment]
Thanks for the PR, won't this break all the other translations though?

**Comment by ClementMR** [comment]
Translations using the old method did not work, but now they are effective

File: mods/other/random_messages/init.lua [hunk 1]
```diff
@@ -34,60 +34,61 @@ end
...
+		S("An alternative method to place markers is by (left/right) clicking while holding the zoom key (default: Z)."),
+		S("Use /mp <player> to send a marker to a specific teammate."),
+		S("Use /msg <playername> <message> to send a private message (PM) to another player."),
+		S("To check someone's league in all modes, use /league <player>"),
+		S("Use /leagues to list leagues and the placement needed to get to them."),
+		S("To place an HP marker, run /mhp or left/right click while holding the zoom key (default: Z) ")..
+			S("and looking at your feet."),
+		S("Use /map to print the current map name and its author."),
+		S("To check someone's ranking in all modes, use /rank mode:all <player_name>"),
+		S("Building a base with materials stronger than cobblestone, such as wood or reinforced cobble, " ..
```
Comment (ClementMR) [inline mods/other/random_messages/init.lua:83]: I lately did something wrong on this line

Comment (ClementMR) [reply mods/other/random_messages/init.lua:83]: > I lately did something wrong on this line

```suggestion
		S("Building a base with materials stronger than cobblestone, such as wood or reinforced cobble, keeps your flag safer."),
```

File: mods/other/random_messages/init.lua [hunk 2]
```diff
@@ -34,60 +34,61 @@ end
...
+			"keeps your flag safer."),
```
Comment (ClementMR) [inline mods/other/random_messages/init.lua:84]:
```suggestion

```

File: mods/other/random_messages/init.lua (hunk 1 above)
Comment (ClementMR) [inline mods/other/random_messages/init.lua:83]: Lately made a mistake

Comment (ClementMR) [inline mods/other/random_messages/init.lua:83]:
```suggestion
		S("Building a base with materials stronger than cobblestone, such as wood or reinforced cobble, keeps your flag safer."),
```

File: mods/other/random_messages/init.lua (hunk 2 above)
Comment (ClementMR) [inline mods/other/random_messages/init.lua:84]:
```suggestion
```

**Review by ClementMR — COMMENTED** [review]
> I lately did something wrong

**Pull request merged**